### DISCUSSION TITLE
S330 cameras - Failed to set up error message - running Home Assistant Core and docker compose

### DISCUSSION THREAD

**Discussion by CarstenDennis** [fuatakgun/eufy_security#1011] [General]
Hi

I get this error:
![image](https://github.com/fuatakgun/eufy_security/assets/139639309/ff71d77a-47e1-44c8-84d7-c246096e293c)

(127.0.0.1 no devices or entities ! failed to set up)
My config is this
![image](https://github.com/fuatakgun/eufy_security/assets/139639309/a5bfd642-60b9-4fa5-b3ce-8763927f53e3)
![image](https://github.com/fuatakgun/eufy_security/assets/139639309/572d713f-db44-4dca-bf34-ef8adcdd5656)

I've installed the Eufy Security integration and got the 2 containers running with this config (left out username and pw)

![image](https://github.com/fuatakgun/eufy_security/assets/139639309/89e8bb29-50db-405a-bc21-bc65d75a4067)
![image](https://github.com/fuatakgun/eufy_security/assets/139639309/979997c3-574a-4c09-8c95-db45c0c178ef)

I ofcause checked that the account used has access to the camers.

Does anyone have the slightest idea what I'm missing here?

Thanks!

**Comment by fuatakgun** [comment]
You are using a very old version of the docker image, move to the latest one

**Comment by CarstenDennis** [comment]
Ow, that might explain it.

Can you tell me what the latest one is? 
I thought it was that one.

**Reply by jellejans** [reply]
just replace :1.3.0 with :latest

**Comment by CarstenDennis** [comment]
I replaced 1.3.0 with :latest as Jellejans wrote - thanks!

The container was recreated and I've restated Home Assistant.

I've not gotten any devices or entries though,  it looks like this:

![image](https://github.com/fuatakgun/eufy_security/assets/139639309/a189a933-2600-415d-abff-354f1a3f7538)

I have logged in on a secondary phone with the Eufy credentials, that I also defined in the config file - and the phone access works, there is no 2 factor authentication.

Am I missing something else completly simple?

**Reply by CarstenDennis** [reply]
Note, the logfile is saying this
26.11.2023
17.34.44
2023-11-26 16:34:44.044  INFO  No stations found. 
26.11.2023
17.34.44
2023-11-26 16:34:44.514  INFO  No devices found. 
26.11.2023
17.44.45
2023-11-26 16:44:45.498  INFO  No stations found. 
26.11.2023
17.44.45
2023-11-26 16:44:45.970  INFO  No devices found. 
26.11.2023
17.54.46
2023-11-26 16:54:46.974  INFO  No stations found. 
26.11.2023
17.54.47
2023-11-26 16:54:47.456  INFO  No devices found. 

I logged my secondary phone out - and confirmed that the credentials are working.

I've also put the cameras to low streaming resolution and restarted the homebase - still with no change.

Any ideas?

**Reply by alrassia** [reply]
Did you check if there are any errors in the log of the Eufy-ws container?

**Reply by fuatakgun** [reply]
I would say, you have the wrong country code :-) so add-on is logging into wrong region

**Reply by CarstenDennis** [reply]
fuatakgun is correct - I haden't changed it in the docker-compose file, so default was US :-)

Now I can see the cameras and homebase in HA - thanks! I'll close the thread
